Content.Server/GameObjects/EntitySystems/BuckleSystem.cs

```diff
@@ -31,9 +37,15 @@ public override void Shutdown()
         {
             base.Shutdown();
 
-            UnsubscribeLocalEvent<MoveEvent>();
-            UnsubscribeLocalEvent<EntInsertedIntoContainerMessage>();
-            UnsubscribeLocalEvent<EntRemovedFromContainerMessage>();
+            UnsubscribeLocalEvent<BuckleComponent, MoveEvent>();
+
+            UnsubscribeLocalEvent<StrapComponent, RotateEvent>();
+
+            UnsubscribeLocalEvent<BuckleComponent, EntInsertedIntoContainerMessage>();
+            UnsubscribeLocalEvent<StrapComponent, EntInsertedIntoContainerMessage>();
+
+            UnsubscribeLocalEvent<BuckleComponent, EntRemovedFromContainerMessage>();
+            UnsubscribeLocalEvent<StrapComponent, EntRemovedFromContainerMessage>();
 
             UnsubscribeLocalEvent<BuckleComponent, AttackHandEvent>(HandleAttackHand);
```

Comment:
you dont need to unsubscribe anymore? or am i mixing things up? @Zumorica

Reply:
You indeed don't need to anymore

Reply:
dealt with